src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -100,7 +100,7 @@ public void AnalyzeCommandBase_RootContextIsDisposed()
             }
 
             ITestAnalyzeCommand command = multithreaded
-                ? (ITestAnalyzeCommand)new TestMultithreadedAnalyzeCommand()
+                ? new TestMultithreadedAnalyzeCommand()
                 : (ITestAnalyzeCommand)new TestAnalyzeCommand();
```

Comment:
>(ITestAnalyzeCommand)new TestAnalyzeCommand() [](http://example.com/codeflow?start=18&length=45)

cant you apply the same in this line?

Reply:
That one can not be removed, will have compile error:

Error CS8957 
Conditional expression is not valid in language version 8.0 because a common type was not found between 'TestMultithreadedAnalyzeCommand' and 'TestAnalyzeCommand'. 
To use a target-typed conversion, upgrade to language version 9.0 or greater. 
Test.UnitTests.Sarif.Driver (netcoreapp3.1) C:\GH\s-s\src\Test.UnitTests.Sarif.Driver\Sdk\AnalyzeCommandBaseTests.cs 102